apps/web/src/server/views/appeals/appeal/appeal-details.njk

```diff
@@ -306,72 +307,52 @@
 					text: "Case documentation"
 				},
 				content: {
-					html: govukSummaryList({
-						rows: [{
-							key: {
-								text: "Appellant case"
-							},
-							value: {
-								html: statusTag(appeal.documentationStatus.appellantCase)
-							},
-							actions: appeal.documentationStatus.appellantCase | actionsParameterForDocumentStatus("#", "Review", "Appellant case")
-						}, {
-							key: {
-								text: "LPA questionnaire"
-							},
-							value: {
-								html: statusTag(appeal.documentationStatus.lpaQuestionnaire)
-							},
-							actions: appeal.documentationStatus.lpaQuestionnaire | actionsParameterForDocumentStatus("#", "Review", "LPA questionnaire")
-						}, {
-							key: {
-								text: "Statement and Proofs"
-							},
-							value: {
-								html: statusTag(appeal.documentationStatus.statementAndProofs)
-							},
-							actions: appeal.documentationStatus.statementAndProofs | actionsParameterForDocumentStatus("#", "Review", "Statement and Proofs")
-						}, {
-							key: {
-								text: "Interested parties"
-							},
-							value: {
-								html: statusTag(appeal.documentationStatus.interestedParties)
+					html: govukTable({
+						firstCellIsHeader: true,
+						head: [
```

Comment:
The table header doesn't show a horizontal bar when it is a child of an accordion. This is because of some css found here : web>src>styles> 7-components> _table.scss
`.govuk-accordion__section-content {
  .govuk-summary-list:last-child {
	margin-bottom: 25px;
  }

  .govuk-table__row:last-child {
	.govuk-table__header,
	.govuk-table__cell {
	  border-bottom: none;
	}
  }
}`

This may need to be addressed as the table will look different from the design otherwise.

Reply:
Discussed, this is a separate ticket: https://pins-ds.atlassian.net/jira/software/projects/BOAT/issues/BOAT-164